packages/flutter/lib/src/material/button.dart

```diff
@@ -194,29 +196,28 @@ class _RawMaterialButtonState extends State<RawMaterialButton> {
         ),
       ),
     );
-    BoxConstraints constraints;
+    double minWidth;
```

Comment:
There's nothing wrong with what you've got here but:  you could define a minSize var and then assign it Size.zero or const Size(48.0, 48.0) and extend that to _ButtonPadding and etc.

Reply:
Done